BigDecimal.js

```diff
@@ -507,7 +508,7 @@ BigDecimal.prototype.toString = function () {
   const s = x.significand.toString();
   const E = sum(x.exponent, s.length - 1);
   const e = typeof E === 'number' ? E : Number(BigInt(E));
-  const significand = +s.charCodeAt(s.length - 1) === '0'.charCodeAt(0) ? (s.replace(/0+$/g, "") || "0") : s;
+  const significand = s.charCodeAt(s.length - 1) === ZERO_CHAR ? (s.replace(/0+$/g, "") || "0") : s;
```

Comment:
I need "+" as I am using https://github.com/Yaffle/babel-plugin-transform-bigint to support bigints in old browsers
ZERO_CHAR is also should not give much